src/pspm_cfg/pspm_cfg_glm_sps.m

```diff
@@ -77,19 +77,19 @@
 chan_def_left         = cfg_const;
 chan_def_left.name    = 'Last left eye';
 chan_def_left.tag     = 'chan_def_left';
-chan_def_left.val     = {'pupil_l'};
+chan_def_left.val     = {'sps_l'};
 chan_def_left.help    = {'Use last left eye channel.'};

```

Comment:
could you please change the help text as well? 
'use the last sps channel from left eye'